### ISSUE TITLE
URIs from GTFS are escaped

### ISSUE THREAD

**Issue by Bertware** [julianrojas87/linked-connections-server#19]
When the original GTFS file contains URIs, then those URIs are escaped when the GTFS is converted into Linked Connections.

The following configuration

```
{
    "companyName": "dsb",
    "keywords": ["Train", "Stops"],
    "geographicArea": "http://sws.geonames.org/2623032",
    "downloadUrl": "./gtfs/DSB.zip",
    "downloadOnLaunch": false,
    "updatePeriod": "0 0 0 0 1 *",
    "fragmentSize": 500000,
    "realTimeData": false,
    "baseURIs": {
    "stop": "{connection.departureStop}",
    "connection": "http://dk.lc.bertmarcelis.be/connections/dsb/{connection.departureTime(YYYYMMDD)}/{connection.$
    "trip": "http://dk.lc.bertmarcelis.be/vehicle/dsb/{trips.trip_id}/{connection.departureTime(YYYYMMDD)}",
    "route": "http://dk.lc.bertmarcelis.be/routes/dsb/{routes.route_id}"
            }
```

results in 

```
 {
      "@id": "http://dk.lc.bertmarcelis.be/connections/dsb/20181123/http%3A%2F%2Fdk.lc.bertmarcelis.be%2F000008600566/55407198",
      "@type": "Connection",
      "departureStop": "http%3A%2F%2Fdk.lc.bertmarcelis.be%2F000008600566",
      "arrivalStop": "http%3A%2F%2Fdk.lc.bertmarcelis.be%2F000008600567",
      "departureTime": "2018-11-23T18:43:00.000Z",
      "arrivalTime": "2018-11-23T18:44:00.000Z",
      "gtfs:trip": "http://dk.lc.bertmarcelis.be/vehicle/dsb/55407198/20181123",
      "gtfs:route": "http://dk.lc.bertmarcelis.be/routes/dsb/11748_2",
      "direction": "Svendborg St.",
      "gtfs:pickupType": "gtfs:Regular",
      "gtfs:dropOffType": "gtfs:Regular"
    },
```

When the departureStop should be `http://dk.lc.bertmarcelis.be/000008600566`.

This is an issue as the GTFS file contains stops from 3 different base URIs, meaning that they have to be defined in the GTFS file. A suggested solution would be to not encode values if they are at the first position in the template (if they are at the position where you'd expect http), or to only encode them if the template starts with http, or to pass the base URIs as a nested object which specifies if encoding should be disabled, e.g. `{"format": "{...}", "escape": false}` .

**Comment by pietercolpaert** [comment]
Related issue: https://github.com/linkedconnections/gtfs2lc/issues/72

Of course a base URI approach will always escape URI parameters. However, we should be able to have other strategies for retrieving the URIs too. Do you have time and ambitions to solve https://github.com/linkedconnections/gtfs2lc/issues/72? If not, I’ll do it when I find time :)

**Comment by julianrojas87** [comment]
Should be addressed in [linkedconnections/gtfs2lc#72](https://github.com/linkedconnections/gtfs2lc/issues/72)
